server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPServer.java

```diff
@@ -35,26 +35,32 @@
 
 public class JMAPServer implements Configurable {
 
+    private static final String DOWNLOAD_URL = JMAPUrls.DOWNLOAD + "/*";
```

Comment:
I would rather define another method on configurationBuilder that handle this `*` stuff